logos/src/logos/responses.py

```
@@ -211,236 +171,141 @@ def proxy_behaviour(headers, providers, path):
+            method="POST",
+            url=forward_url,
+            json=json_data,
+            headers=proxy_headers,
+            timeout=30,
+        )
 
-    system_text = ""
-    user_text = ""
+    try:
+        response: dict = resp.json()
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)